otx/algorithms/classification/configs/efficientnet_b0_cls_incr/selfsl/model.py

```diff
@@ -2,16 +2,23 @@
 
 # pylint: disable=invalid-name
 
-_base_ = "../model.py"
+_base_ = ["../../../../../recipes/stages/classification/selfsl.yaml", "../../base/models/efficientnet.py"]
 
 model = dict(
     type="BYOL",
     task="classification",
+    backbone=dict(
+        version="b0",
+    ),
     base_momentum=0.996,
     neck=dict(type="SelfSLMLP", in_channels=1280, hid_channels=4096, out_channels=256, with_avg_pool=True),
     head=dict(
         type="ConstrastiveHead",
         predictor=dict(type="SelfSLMLP", in_channels=256, hid_channels=4096, out_channels=256, with_avg_pool=False),
+        loss=dict(
+            type="CrossEntropyLoss",
+            loss_weight=1.0,
+        ),
```

Comment:
You can remove this loss arg in self-sl head because loss will be automatically calculated through BYOL loss in the head.

Reply:
And it looks better to insert `_delete_=True` in `head` not to be inherited anything from the previous one.
(It is not essential, but just for making the recipe more clear.)

Reply:
Thanks for the review. I've fixed it, but can you double check? https://github.com/openvinotoolkit/training_extensions/pull/1875/commits/420fb5b81daa29b4237f5f8cc6de1bf874ddf8ae